shared/constants/types/crypto.tsx

```diff
@@ -41,6 +41,8 @@ export type CommonState = {
   errorMessage: HiddenString
   input: HiddenString
   inputType: InputTypes
+  // to ensure what the user types matches the input
+  outputMatchesInput: boolean
```

Comment:
we keep track if the output matches the input. While typing we allow you to see the mismatched values (we want the UX feedback) but the actions (copy/save etc) are disabled